pkgs/development/python-modules/pysychonaut/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, requests, requests-cache, beautifulsoup4 }:
+
+buildPythonPackage rec {
+  pname = "PySychonaut";
+  version = "0.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wgk445gmi0x7xmd8qvnyxy1ka0n72fr6nrhzdm29q6687dqyi7h";
+  };
+
+  preConfigure = ''
+    substituteInPlace setup.py --replace "bs4" "beautifulsoup4"
+  '';
+
+  propagatedBuildInputs = [ requests requests-cache beautifulsoup4 ];
+
+  pythonImportsCheck = [ "pysychonaut" ];
+
+  meta = with lib; {
+    description = "Unofficial python api for Erowid, PsychonautWiki and AskTheCaterpillar";
+    homepage = "https://github.com/OpenJarbas/PySychonaut";
+    maintainers = [ maintainers.ivar ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ ivar ];
```

Reply:
IMO your proposal looks a bit worse, and functionality wise it does not improve anything. I only need one entry of the `maintainers` list, no need for everything to be in scope.

There are no guidelines on this as far as i know, so  i think this should be fine.

Reply:
It makes changing maintainers very easy and repeating the same word 3 times looks worse.

Reply:
I agree, however at the moment it's only repeated twice, in both solutions.

I think that change can be made when it's actually applicable, as of now it has no advantages.